docs/source/technical_tutorials/visualization/visualizing_titiler-pgstac.ipynb

```diff
@@ -0,0 +1,751 @@
+{
```

Comment:
<p>is it confusing that both folium and ipyleaflet use leaflet and that MAAP/Pangeo have no built in Mapbox option (do we even need to mention it?)</p><p> </p><p>should we be leveraging stac_ipyleaflet?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/423/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I can remove the Mapbox note. I only mentioned it because we have to do some work to load the tile layer and its parameters into leaflet but Mapbox can use the tilejson.json out of the box. That detail is probably not relevant in the notebook context, though!</p>